packages/core/src/components/tooltip/tooltip.template.ts

```diff
@@ -2,8 +2,8 @@ import { Tooltip } from '@dso-toolkit/sources';
 import { html } from 'lit-html';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 
-export function tooltipTemplate({ active, position, label, id }: Tooltip) {
-  return html`<dso-tooltip position=${position} for=${ifDefined(id)} ?active=${active}>
+export function tooltipTemplate({ active, descriptive, position, label, id }: Tooltip) {
+  return html`<dso-tooltip ?descriptive=${ifDefined(descriptive)} position=${position} for=${ifDefined(id)} ?active=${active}>
```

Comment:
Voor het zetten van boolean attributes met lit-html hoef je geen `ifDefined()` te gebruiken:

```html
<dso-tooltip ?descriptive=${ietsWatTruthyIs}>`
```

* https://lit.dev/docs/api/directives/#ifDefined
* https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions

Net zoals met `?active=${active}`.

Reply:
We hebben er samen wat leuks van gemaakt.